src/sql/workbench/electron-browser/modelComponents/diffeditor.component.ts

```diff
@@ -79,8 +81,20 @@ export default class DiffEditorComponent extends ComponentBase implements ICompo
 		this.editorUriRight = uri2.toString();
 
 		let cancellationTokenSource = new CancellationTokenSource();
-		let editorinput1 = this._instantiationService.createInstance(UntitledEditorInput, uri1, false, 'plaintext', '', '');
-		let editorinput2 = this._instantiationService.createInstance(UntitledEditorInput, uri2, false, 'plaintext', '', '');
+		this._textModelService.registerTextModelContentProvider('diffEditor', {
```

Comment:
This should be added to the list of disposables (this._register)

Reply:
added